launcher/translations/TranslationsModel.cpp

```diff
@@ -454,6 +454,7 @@ QVariant TranslationsModel::data(const QModelIndex& index, int role) const
                 return QString("%1%").arg(lang.percentTranslated(), 3, 'f', 1);
             }
         }
+        qWarning("TranslationModel::data not implemented when role is DisplayRole");
```

Comment:
Won't this still fall through?

Reply:
I remember testing it and it doesn't seem to cause the compiler warning. I'll check again

Reply:
there are only two entries for that enum. So it won't fallthrough

Reply:
It will if another is added.